makefile

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: GPL-2.0
+
+# Default target executed when no arguments are given to make.
+default_target: all
+
+.PHONY : default_target
+
+# Allow only one "make -f <foo>" at a time, but pass parallelism.
+.NOTPARALLEL:
+
+
+mobile:
+	cd build-mobile; LIBRARY_PATH=../install_root/lib make 
+	cd build-mobile; LIBRARY_PATH=../install_root/lib make install
+
+desktop:
+	cd build; LIBRARY_PATH=../install_root/lib make 
```

Comment:
ditto

Reply:
It is just a convenience for others and me, who constantly compile the 2 versions.

But no big deal if you do not like, I can do the same with an alias.

Reply:
It's not that I don't like it. It's that I wonder who'd use it.
But once it's implemented in a sane way so it actually works on a freshly checked out tree... I'm fine.
You don't necessarily have to call build.sh from here - just don't fail with a weird error if someone calls this without calling build.sh first

Reply:
Corrected